extras/conversion-vm/centos.ks

```diff
@@ -41,4 +39,8 @@ yum install -y http://resources.ovirt.org/pub/yum-repo/ovirt-release42.rpm
 
 echo "Installing ovirt-ansible-v2v-conversion-host package.."
 yum install -y ovirt-ansible-v2v-conversion-host
+
+echo "Downloading virtio-win ISO"
+mkdir -p /usr/share/virtio-win
+curl -vL -o /usr/share/virtio-win/virtio-win.iso 'https://fedorapeople.org/groups/virt/virtio-win/direct-downloads/stable-virtio/virtio-win.iso'
```

Comment:
I'm not sure if curl from the internet is acceptable for OpenStack Image build process, going to ask..

Reply:
This concerns only upstream. For downstream the package comes from channel.

Reply:
Ah, looks good to me then :-1: